src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/DefaultVSLSPTagHelperTooltipFactory.cs

```diff
@@ -415,9 +415,19 @@ static void ClassifyExistingTextRun(List<ClassifiedTextRun> runs, StringBuilder
 
     private static ContainerElement CombineClassifiedTextRuns(IReadOnlyList<DescriptionClassification> descriptionClassifications, ImageElement glyph)
     {
+        var isFirstElement = true;
         var classifiedElementContainer = new List<ContainerElement>();
         foreach (var classification in descriptionClassifications)
         {
+            if (isFirstElement)
```

Comment:
A comment here might be helpful for future readers (e.g. something short like "adds blank lines between multiple classified elements")